identity/handler.go

```diff
@@ -136,7 +136,7 @@ type listIdentitiesParameters struct {
 //	  default: errorGeneric
 func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	page, itemsPerPage := x.ParsePagination(r)
-	is, err := h.r.IdentityPool().ListIdentities(r.Context(), page, itemsPerPage)
+	is, err := h.r.IdentityPool().ListIdentities(r.Context(), ExpandDefault, page, itemsPerPage)
```

Comment:
Consider expand as query parameter here for the handler. Could be used to optimize the list op in the console view

Reply:
Definitely, but will need to track this as a follow up. Right now my focus is on improving performance of the default api request path :)

Reply:
👍 would you be open to create such an issue? :)